Mathlib/Algebra/DirectSum/Decomposition.lean

```diff
@@ -232,6 +251,15 @@ variable (ℳ : ι → Submodule R M)
 
 variable [Decomposition ℳ]
 
+/-- A convenience method to construct a decomposition from an `LinearMap`, such that the proofs
+of left and right inverse can be constructed via `ext`. -/
+abbrev Decomposition.ofLinearMap (decompose : M →ₗ[R] ⨁ i, ℳ i)
```

Comment:
Again, I think I'd place this just above `variable [Decomposition ℳ]` two lines up.